### PR TITLE
Decode a Morse code message

### PR DESCRIPTION
**Pull request by MenendezON**
This pull request concerns the implementation of a function, that allows a user to translate a Morse message into a normal message that could be understood by a human.

### CODE REVIEW

File: message.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        '9' => '----.' ,
+        '0' => '-----' ,
+        ' ' => '/' 
+    }
+
+morse = morse.gsub!("   ", " / ", )
+
+morse.split(' ').each do |el|
+        print morse_code.key(el).to_s
+end (edited)
```
Comment (descholar-ceo) [inline message.rb]: - [ ] Could you remove `(edited)` it's causing your program to crash when we run it!
```suggestion
end 
```

File: README.md [hunk 2]
```diff
@@ -1 +1,110 @@
...
+
+- Github: [@NalianAchei2020](https://github.com/NalianAchei2020)
+- Linkedin: [Nalian Achei](https://www.linkedin.com/in/nalian-achei-683208275)
+- Twitter: [@NalianAchei](https://twitter.com/NalianAchei?t=E3ePLcJ7B45dBa8SBFIXDg&s=09)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ]
```
Comment (descholar-ceo) [inline README.md]: - [ ] Don't leave empty spaces in your readme file, in order to make sure that your repository is professional always
- [ ] You need to list all future features, it's a hard requirement for this project

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @MenendezON && @NalianAchei2020 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is:
- [ ] Failing to run due to the issue I listed down inline you codes, 
- [ ] Unprofessional readme file which misses some important points [kindly check inline your readme file down], 
- [ ] Doesn't have linters installed, kindly install rubocop linters before requesting a review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @TEAM 👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉


## Highlight 👍 
✅ solved basic coding challenge on how to decode a Morse code message.

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**